resources/sdk/pureclouddotnet/templates/README.mustache

```diff
@@ -144,6 +143,27 @@ Once an access token has been obtained from one of the OAuth methods, it must be
 PureCloudPlatform.Client.V2.Client.Configuration.Default.AccessToken = "BL4Cb3EQIQFlqIItaj-zf5eIhAiP96zk3333QImd24P99ojbFHtpgUTJdRIkuUYfXMy0afEnZcWnEQ";
 ```
 
+#### Setting the max retry time
+
+By default, the .NET SDK does not automatically retry any failed requests.
+To enable automatic retries, provide a RetryConfiguration object with the maximum number of seconds to retry requests when building the ApiClient instance.
+
+Building a `RetryConfiguration` instance:
+```{"language":"csharp"}
+ var retryConfig = new ApiClient.RetryConfiguration
+            {
```

Comment:
Use indention of 2 spaces per level in the readme

Reply:
fixed